### ISSUE TITLE
Audio block: Improve focus style

### ISSUE THREAD

**Issue by carolinan** [WordPress/twentytwentyone#207]
**Describe the bug**
When the audio block is added, it looks different in different browsers as expected, but the outer focus style in all browsers needs to be more visible.

In Firefox, there is no spacing between the element and the thin dotted outline so the outline is difficult to see.
In chrome, the focus is thicker, but there needs to be more spacing, see screenshot below.

**To Reproduce**
1. Add an audio block. There is an audio file in the [Theme Unit Test](https://github.com/WPTT/theme-unit-test) if needed
2. View the front
3. Place focus on the block

**Screenshots**
Chrome at the top, Firefox below
![Focus styles for the audio block are not visible enough](https://user-images.githubusercontent.com/7422055/94805064-11c1a280-03ec-11eb-860f-666c810599da.png)

**Comment by melchoyce** [comment]
Solid focus style:

<img src="https://user-images.githubusercontent.com/2846578/94848619-40db1280-03f2-11eb-935e-5681770e0c4a.png" width="600" />
<img src="https://user-images.githubusercontent.com/2846578/94848687-5bad8700-03f2-11eb-94a7-7c6d1ccd191c.png" width="600" />

```
outline-offset: 5px;
outline: 2px solid var(--global--color-border);
```

Alternately, dotted like buttons:

<img src="https://user-images.githubusercontent.com/2846578/94848758-754ece80-03f2-11eb-80a1-8f8bb67c004e.png" width="600" />
<img src="https://user-images.githubusercontent.com/2846578/94848778-7bdd4600-03f2-11eb-9dfe-a98c7afaefd8.png" width="600" />

```
outline-offset: 5px;
outline: 2px dotted var(--global--color-border);
```

(Unsure if border is the right variable here, just knew it would get the right color when testing)
